### ISSUE TITLE
Accuracy degradation in TensorRT 10.0.1.6 when running SigLIP on a 4070ti GPU with FP16

### ISSUE THREAD

**Issue by VictorBebnev** [NVIDIA/TensorRT#3908]
I ran SigLIP with FP16 and got degraded accuracy for TRT 10.0.1.6.
Precision on TRT 8.6.1:      **71,6**
Precision on TRT 10.0.1.6: **68,4**
Diff: **3,2**

GPU: 4070Ti
CUDA: 12.1
NVIDIA Driver: 550.54.14
ONNX OP set: 17
Torch: 2.1.2
Conversion pipeline: PyTorch -> ONNX -> TRT

How to load model 
```
import open_clip
import torch

model = open_clip.create_model_and_transforms('ViT-B-16-SigLIP-512', pretrained='webli')[0]
torch.onnx.export(
      model.visual,
      torch.zeros(1, 3, 512, 512),
      "test.onnx",
      opset_version=17,
      input_names=["input"],
      output_names=["output"],
      dynamic_axes={"input": {0: "batch"}, "output": {0: "batch"}},
)
```

Export
```
trtexec --onnx=test.onnx --saveEngine=test.engine --minShapes=input:1x3x512x512 --maxShapes=input:100x3x512x512 --optShapes=input:30x3x512x512 --memPoolSize=workspace:9000 --fp16
```

**Comment by aliencaocao** [comment]
Hi how did you do inference? Im trying to convert hugging face's siglip to TRT too
